_examples/pubsub/pubsub.go

```diff
@@ -117,6 +118,7 @@ func publish(sessions chan chan session, messages <-chan message) {
 			select {
 			case confirmed, ok := <-confirm:
 				if !ok {
+					pub.Close()
```

Comment:
Won't it be closed anyway here? https://github.com/rabbitmq/amqp091-go/blob/main/_examples/pubsub/pubsub.go#L135

Reply:
I don't think so, because the `break Publish` exits the `for` loop.